htdocs/comm/propal/card.php

```diff
@@ -2343,6 +2344,10 @@
 
 			$parameters = array();
 			$reshook = $hookmanager->executeHooks('formAddObjectLine', $parameters, $object, $action); // Note that $action and $object may have been modified by hook
+		}elseif($action == 'editline')
```

Comment:
why 
elseif($action == 'editline') 
when if is
 if ($action != 'editline')

else is enough